src/makeRobotAccountant.js

```diff
@@ -4,9 +4,23 @@
  *
  * @return {function}
  */
+const ERROR_MESSAGE = 'Bzzz... Error!';
+const USAGE_LIMIT = 3;
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
```

Comment:
It would be cool to specify exactly which counter it is.
```suggestion
  let operationsCount = 0;
```